### ISSUE TITLE
Convenient constructor

### ISSUE THREAD

**Issue by wagdav** [wagdav/dimarray#3]
Implement a convenient constructor, like:
     a = DimArray(rand(10), t=arange(10))

**Comment by KornelJahn** [comment]
I doubt it's possible, since **kwargs is a non-ordered dictionary and ordered dimensions are required.

**Comment by wagdav** [comment]
Good point. Though for arrays where array.ndim == 1 would make sense.

Or 

```
a = dimarray(rand(10), t=arange(10), x=arange(10), order='xt')
```

Where dimarray is a convenience function to create DimArray objects.

**Comment by KornelJahn** [comment]
With an
    order = tuple(order)
inside the implementation, 'cause range names are not necessarily single characters. This way, your notation would work, but also order=('rho', 'phi').

**Comment by wagdav** [comment]
Any iterable type would do. 

'xy' => 'x', 'y'

('rho','phi') => 'rho', 'phi'

**Comment by wagdav** [comment]
Implemented in 547d943b563eb516ee50e6d3d3ba8d34f209d1b5.
